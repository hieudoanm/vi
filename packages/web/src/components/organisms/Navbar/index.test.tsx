import { UserProvider } from '@auth0/nextjs-auth0';
import { render } from '@testing-library/react';
import NavBar from '.';

describe('NavBar', () => {
  test('render default', () => {
    const wrapper = render(
      <UserProvider>
        <NavBar />
      </UserProvider>
    );
    expect(wrapper.container).toMatchSnapshot();
  });
});
