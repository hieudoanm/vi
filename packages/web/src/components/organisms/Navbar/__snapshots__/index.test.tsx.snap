// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar render default 1`] = `
<div>
  <nav
    class="md:py-6 py-4 w-full fixed z-10 bg-cyan-50 transition-all css-m0z1ly"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="flex justify-between items-center"
      >
        <div
          class="font-extrabold tracking-widest text-black"
        >
          VI
        </div>
      </div>
    </div>
  </nav>
</div>
`;
